src/theme/components/progressIndicator.ts

```diff
@@ -44,13 +50,17 @@ const ProgressIndicator = {
         // "Progress" as the key name in index.ts
         "> div": {
           bg: darkMode ? "ui.white" : "ui.link.primary",
+          _dark: "dark.ui.focus",
```

Comment:
I think you need the key attribute `{ outline: "dark.ui.focus" }` instead of just adding the value to `_dark`.